openpower/package/machine-xml/machine-xml.mk

```diff
@@ -91,6 +91,16 @@ define MACHINE_XML_BUILD_CMDS
             $(PETITBOOT_BIOS_XML_METADATA_FILE) \
             $(PETITBOOT_XSLT_FILE) \
             $(BIOS_XML_METADATA_FILE)
+
+        # Create the wofdata
+        if [ -e $(MRW_HB_TOOLS)/wof-tables-img ]; then \
+            chmod +x $(MRW_HB_TOOLS)/wof-tables-img; \
+        fi
+
+        if [ -e $(MRW_SCRATCH)/wof_v4_dd1_export.csv ]; then \
```

Comment:
The filename of the csv should be more generic, this name has a version and dd level in there.  The revision will change a lot.  I think we have 2 options:
a) use a common generic name for everything - wofdata.csv?
b) create a variable to use a machine-specific filename, e.g. wofdata_witherspoon.csv